index.js

```diff
@@ -433,6 +541,33 @@ function toggleSidebar() {
 }
 
 /**
+* Shows the user data sidebar which provides the user with a button to
+* open the newtab page, reset the auto-import, and view a preview of the data that was imported
+*/
+function showUserDataSidebar() {
+    store('lastSidebarLaunchTime', Date.now());
+
+    content = sidebar.Sidebar({
+        id: 'allboard-importdata',
+        title: 'We\'ve got your back.',
+        url: './tmpl/show_data.html',
+        onAttach: function(worker) {
+            getHistory(worker);
+            getBookmarks(worker);
```

Comment:
These two functions has a lot in common, after out first deadline, we should look at creating one utility function that can be used by both, passing only what is unique to the utility.